tests/xtro-sharpie/EnumCheck.cs

```diff
@@ -189,15 +192,16 @@ public override void VisitEnumDecl (EnumDecl decl, VisitKind visitKind)
 					if ((value.InitExpr != null) && value.InitExpr.EvaluateAsInt (decl.AstContext, out var integer))
 						n = integer.SExtValue;
 
-					native_signed_values [n] = value.ToString ();
+					native_signed_values [n] = new (value.ToString (), value.IsDeprecated ());
 					// assume, sequentially assigned (in case next `value.InitExpr` is null)
 					n++;
 				}
 
 				foreach (var value in native_signed_values.Keys) {
-					if (!managed_signed_values.ContainsKey (value))
-						Log.On (framework).Add ($"!missing-enum-value! {type.Name} native value {native_signed_values [value]} = {value} not bound");
-					else
+					if (!managed_signed_values.ContainsKey (value)) {
+						if (!native_signed_values [value].Deprecated && !decl.IsDeprecated ())
```

Comment:
why not a single if?

Reply:
That changes the logic - we'll hit the else branch below in more cases.